src/EditorFeatures/Core/EditorConfigSettings/Data/CodeStyle/CodeStyleSetting.PerLanguageEnumCodeStyleSetting.cs

```diff
@@ -37,16 +37,14 @@ private class PerLanguageEnumCodeStyleSetting<T> : EnumCodeStyleSettingBase<T>
 
             protected override void ChangeSeverity(NotificationOption2 severity)
             {
-                var option = GetOption();
-                option.Notification = severity;
-                _ = Updater.QueueUpdateAsync(_option, option);
+                ICodeStyleOption option = GetOption();
```

Comment:
var?

Reply:
Not possible. The next line is using an explicitly implemented interface method.